### PR TITLE
Aether-3395 Part b - update list views

### PR DESCRIPTION
**Pull request by SeanCondon**

### CODE REVIEW

File: src/app/enterprise.service.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import { Injectable } from '@angular/core';
+import { TargetsNames } from '../openapi3/top/level/models';
+import { ApiService as TopLevelApiService } from '../openapi3/top/level/services';
+import { tap } from 'rxjs/operators';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class EnterpriseService {
+    enterprises: TargetsNames;
```
Comment (teone) [inline src/app/enterprise.service.ts:16]: I'm not a huge supported of directly accessing class property, maybe making this private/protected and using a `getter` would provide better isolation?

A pattern I found useful in the past was to always call a `load` (or similar) methods, that would implement a cache if needed but always return a `promise/observable`, that way the "client" doesn't need to know whether the data is already loaded or not.

File: src/app/enterprise.service.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    constructor(private aetherService: TopLevelApiService) {}
+
+    loadTargets() {
+        this.aetherService
+            .targetsTopLevel()
+            .pipe(tap((e) => console.log('Loaded enterprises', e)))
+            .subscribe((enterprises) => (this.enterprises = enterprises));
+    }
+
+    get isSingleton(): boolean {
```
Comment (teone) [inline src/app/enterprise.service.ts:27]: Doesn't look like this method is used anywhere, what is it needed for?

File: src/app/aether-application/application/application-datasource.ts [hunk 3]
```diff
@@ -13,106 +12,90 @@ import {
...
-                                                            ]
-                                                        ) {
-                                                            app[ISINUSE] =
-                                                                'true'; // Any match will set it
-                                                        }
-                                                    }
-                                                );
+        dataSourceObservable.pipe(skipWhile((x) => x === undefined)).subscribe(
+            (appList: ApplicationList) => {
+                appList.forEach((app: Application) => {
```
Comment (teone) [inline src/app/aether-application/application/application-datasource.ts:52]: This is most likely for a different patch, but should we look into generalizing this method to highlight the deleted items as we did for `merge` method to combine data in the basket with data from the APIs?

**Comment by SeanCondon** [comment]
retest this please

File: src/app/aether-application/application/application-datasource.ts (hunk 3 above)
Comment (SeanCondon) [reply src/app/aether-application/application/application-datasource.ts:52]: yeah a different patch - this one is too big already

File: src/app/enterprise.service.ts (hunk 2 above)
Comment (SeanCondon) [reply src/app/enterprise.service.ts:27]: I was going to use it to hide the `enterprise` column when a user was logged in, with access to only a single enterprise

File: src/app/enterprise.service.ts (hunk 1 above)
Comment (SeanCondon) [reply src/app/enterprise.service.ts]: I added a getter

**Review by teone — APPROVED** [review]
@SeanCondon looks good to me, feel free to merge as soon as Jenkins is happy

**Pull request merged**